src/main/resources/static/script.js

```diff
@@ -1,10 +1,41 @@
-let centerLatitude = 41.85;
-let centerLongitude = -87.65;
-let centerZoom = 7;
+
+var map;
+let centerLatitude = 37.85;
+let centerLongitude = -97.65;
+let centerZoom = 4;
 
 function initMap() {
+   //autocomplete creation below. could this be a function that is called first in initmap? to do so, would need to declare variables at top?
+   var originInput = document.getElementById("originInput");
+   var origin = new google.maps.places.Autocomplete(originInput,
+   {
+     componentRestrictions: {'country': ['us']},
+     fields: ['geometry', 'name', 'formatted_address']
+   });
+   var destinationInput = document.getElementById("destinationInput");
+   var destination = new google.maps.places.Autocomplete(destinationInput,
+   {
+    componentRestrictions: {'country': ['us']},
+    fields: ['geometry', 'name', 'formatted_address'] //is there anything else we need? we'll want to create a location object with this info
+   });
+   //handle when a user hits enter upon finding selected place
```

Comment:
can we remove unnecessary comments from this file? line 8, 19, 21

Reply:
resolved